express-notes-api/index.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const dataJson = require('./data.json');
+const notes = dataJson.notes;
+
+app.use(express.json());
+
+// request all notes:
+app.get('/api/notes', (req, res) => {
+  const notesArray = [];
+
+  for (const curNote in notes) {
+    notesArray.push(notes[curNote]);
+  }
+
+  res.status(200).json(notesArray);
```

Comment:
Status `200` is the default so you don't need to set the status if you want it to be `200`